testing/bdd/features/handover/handover.feature

```diff
@@ -60,25 +82,36 @@ Feature: Handover algorithm
     And doc1 is added to the sequence
     And we create a new document
     And it has sequenceNumber 1
-    And it does not have element authorsGroupIdentifier
+    And it does not have attribute authorsGroupIdentifier
     Then doc2 is added to the sequence
 
     Examples:
     | authorsGroupIdentifier_1 | 
     | 1                        | 
 
 
+
   # SPEC-CONFORMANCE: R121
   # The Handover Manager shall not emit any documents derived from documents that do not contain both the parameters 
   # ebuttp:authorsGroupIdentifier and ebuttp:authorsGroupControlToken.
   Scenario: Documents with missing group identifier and token
     Given an xml file <xml_file>
     And a handover node
-    When it does not have element authorsGroupControlToken 
-    And it does not have element authorsGroupIdentifier 
+    When it does not have attribute authorsGroupControlToken 
+    And it does not have attribute authorsGroupIdentifier 
     Then the document is not added to sequence
```

Comment:
maybe this should say `Then the handover node does not emit a derived document` to be precise?